src/sentry/db/models/manager.py

```diff
@@ -86,6 +88,9 @@ def _get_cache(self):
     def _set_cache(self, value):
         self.__local_cache.value = value
 
+    def _generate_cache_version(self):
+        return crc32('&'.join(f.attname for f in self.model._meta.fields))
```

Comment:
Might wanna sort these so they're guaranteed consistent order. Not sure of a case where the order of `_meta.fields` would change though. Likely shouldn't be a concern.